alpine-server/src/main/java/alpine/server/auth/LdapConnectionWrapper.java

```diff
@@ -212,9 +212,7 @@ public List<String> searchForUserName(final DirContext dirContext, String userNa
     public List<String> search(final DirContext dirContext, final String filter, final String searchTerm) throws NamingException {
         LOGGER.debug("Searching / filter: " + filter + " searchTerm: " + searchTerm);
         final List<String> entityDns = new ArrayList<>();
-        final String[] attributeFilter = {};
         final SearchControls sc = new SearchControls();
-        sc.setReturningAttributes(attributeFilter);
```

Comment:
Again, not familiar with LDAP. But could this have a performance impact? 

As per [documentation](https://docs.oracle.com/javase/7/docs/api/javax/naming/directory/SearchControls.html#getReturningAttributes()), passing in `{}` would be saying "return no attributes" whereas the default / `null` would be "return all attributes". Is there a common email attribute that we can pass here instead?

Reply:
According Steven (https://github.com/DependencyTrack/dependency-track/issues/2320#issuecomment-1361666943) it should have been a NULL anyway and likely was a mistake. I don't think there are that many attributes that a performance impact should be noticeable. But we are not really familiar with LDAP either :(

Reply:
Ah I see, sorry I should've checked that issue before making a comment.